<?xml version="1.0" encoding="utf-8"?>
<ui version="4.0">
 <author>
    SPDX-FileCopyrightText: none
    SPDX-License-Identifier: GPL-3.0-or-later
  </author>
 <class>KisWdgOptionsPNG</class>
 <widget class="QWidget" name="KisWdgOptionsPNG">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>561</width>
    <height>266</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PNG Options</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item column="0" row="0">
    <layout class="QGridLayout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <property name="horizontalSpacing">
      <number>0</number>
     </property>
     <property name="verticalSpacing">
      <number>6</number>
     </property>
     <item column="1" row="6">
      <widget class="QCheckBox" name="chkSRGB">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;PNG&lt;/span&gt; files have &lt;span style=&quot; font-style:italic;&quot;&gt;two&lt;/span&gt; options to save &lt;span style=&quot; font-weight:600;&quot;&gt;sRGB&lt;/span&gt; information: as a tag or as an explicit profile. For use with in websites, &lt;span style=&quot; font-style:italic;&quot;&gt;disable&lt;/span&gt; this option. For interchange with other applications, &lt;span style=&quot; font-style:italic;&quot;&gt;enable&lt;/span&gt; this option.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>Embed sRGB profile</string>
       </property>
      </widget>
     </item>
     <item column="1" row="11">
      <widget class="KisColorButton" name="bnTransparencyFillColor">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Background color to replace transparent pixels with.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item column="1" row="8">
      <widget class="QCheckBox" name="alpha">
       <property name="toolTip">
        <string>Disable to get smaller files if your image has no transparency</string>
       </property>
       <property name="whatsThis">
        <string>&lt;p&gt;The Portable Network Graphics (PNG) file format allows transparency in your image to be stored by saving an alpha channel.
You can uncheck the box if you are not using transparency and you want to make the resulting file smaller .&lt;br&gt;Always saving the alpha channel is recommended.&lt;/p&gt;</string>
       </property>
       <property name="text">
        <string>Store alpha channel (transparency)</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item column="1" row="10">
      <widget class="QCheckBox" name="chkAuthor">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Save author nickname and the first contact information of the author profile into the png, if possible.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>Sign with author data</string>
       </property>
      </widget>
     </item>
     <item column="1" row="7">
      <widget class="QCheckBox" name="chkForceSRGB">
       <property name="text">
        <string>Force convert to sRGB</string>
       </property>
      </widget>
     </item>
     <item column="2" row="0">
      <widget class="QLabel" name="textLabel4_2">
       <property name="whatsThis">
        <string>&lt;p&gt;Adjust the compression time. Better compression takes longer.
&lt;br&gt;Note: the compression level does not change the quality of the result.&lt;/p&gt;</string>
       </property>
       <property name="text">
        <string comment="The compression will take a long time; saving the file will be slow.">Small File Size</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item column="1" row="9">
      <widget class="QCheckBox" name="chkMetaData">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Store information like keywords, title and subject and license, if possible.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>Store Metadata</string>
       </property>
      </widget>
     </item>
     <item column="1" row="3">
      <widget class="QCheckBox" name="tryToSaveAsIndexed">
       <property name="toolTip">
        <string>Indexed PNG images are smaller. If you enabled this option, your image will be analyzed to see whether it is possible to save as an indexed PNG.</string>
       </property>
       <property name="text">
        <string>Save as indexed PNG, if possible</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item column="0" row="1" rowspan="2">
      <widget class="QLabel" name="textLabel1">
       <property name="toolTip">
        <string>Note: the compression level does not change the quality of the result</string>
       </property>
       <property name="whatsThis">
        <string>&lt;p&gt;Adjust the compression time. Better compression takes longer.
&lt;br&gt;Note: the compression level does not change the quality of the result.&lt;/p&gt;</string>
       </property>
       <property name="text">
        <string>Compression (Lossless): </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
       </property>
      </widget>
     </item>
     <item column="0" row="11">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Transparent color: </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item column="1" row="0">
      <widget class="QLabel" name="textLabel3">
       <property name="whatsThis">
        <string>&lt;p&gt;Adjust the compression time. Better compression takes longer.
&lt;br&gt;Note: the compression level does not change the quality of the result.&lt;/p&gt;</string>
       </property>
       <property name="text">
        <string comment="The file will save fast but will not be compressed a lot">Large file size</string>
       </property>
      </widget>
     </item>
     <item colspan="2" column="1" row="1">
      <widget class="KisDoubleSliderSpinBox" name="compressionLevel" native="true">
       <property name="toolTip">
        <string>Note: the compression level does not change the quality of the result</string>
       </property>
       <property name="whatsThis">
        <string>&lt;p&gt;Adjust the compression time. Better compression takes longer.
&lt;br&gt;Note: the compression level does not change the quality of the result.&lt;/p&gt;</string>
       </property>
      </widget>
     </item>
     <item column="1" row="4">
      <widget class="QCheckBox" name="interlacing">
       <property name="toolTip">
        <string>Use interlacing when publishing on the Internet.</string>
       </property>
       <property name="whatsThis">
        <string>&lt;p&gt;Interlacing is useful if you intend to publish your image on the Internet.&lt;br&gt;
Enabling interlacing will cause the image to be displayed by the browser even while downloading.&lt;/p&gt;</string>
       </property>
       <property name="text">
        <string>Interlacing</string>
       </property>
      </widget>
     </item>
     <item column="1" row="5">
      <widget class="QCheckBox" name="chkSaveAsHDR">
       <property name="text">
        <string>Save as HDR image (Rec. 2020 PQ)</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item column="0" row="6">
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>414</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KisColorButton</class>
   <extends>QPushButton</extends>
   <header>kis_color_button.h</header>
  </customwidget>
  <customwidget>
   <class>KisDoubleSliderSpinBox</class>
   <extends>QWidget</extends>
   <header>kis_slider_spin_box.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>